tests/ops/op_math/test_controlled_ops.py

```diff
@@ -695,215 +693,6 @@ def test_eigvals_tf(self, tol, op, params, expected_eigvals):
         )
 
 
-def _dot_broadcasted(a, b):
```

Comment:
Mind reminding me why these tests are getting removed?

Reply:
They are all moved to `test_controlled.py`, as parameters in the `TestDecomposition` test class